[docs] [#6515] Add steps to setup Grafana dashboard for macOS (#8924)
yugabyte/yugabyte-db: docs/content/latest/explore/observability/grafana-dashboard/macos.md

  This tutorial uses the [yugabyted](https://docs.yugabyte.com/latest/reference/configuration/yugabyted) local cluster management utility. 
  The purpose of this tutorial is to utilize the Prometheus metrics obtained from following [Prometheus Integration](https://docs.yugabyte.com/latest/explore/observability/prometheus-integration/macos/) and then  creating dashboards via Grafana.
  ## Prerequisite

  A local instance of Prometheus displaying the metrics of your YugabyteDB cluster is required. To do this,
  - Create a local instance of Prometheus and perform  the Steps 1-4 listed under [Prometheus Integration](https://docs.yugabyte.com/latest/explore/observability/prometheus-integration/macos/). (Step 5 is optional)

  ## 1. Setup Grafana and add Prometheus as a data source.
  - Install Grafana and start the server from the [Grafana documentation](https://grafana.com/docs/grafana/latest/installation/mac/).
  - Open the Grafana UI on http://localhost:3000. The default login is `admin` / `admin`. 

>   - Follow the steps to create a Prometheus data source from this [page](https://prometheus.io/docs/visualization/grafana/). 

Reviewer: ```suggestion
  - Follow the steps on [this page](https://prometheus.io/docs/visualization/grafana/) to create a Prometheus data source.
```

  ## 2. Create a dashboard

>   There are different ways to create a dashboard in Grafana. For this tutorial , you will need to use the import option and pick the Yugabytedb community dashboard present under the list of Grafana [Dashboards](https://grafana.com/grafana/dashboards/12620).

Reviewer: ```suggestion

There are different ways to create a dashboard in Grafana. For this tutorial, you'll use the import option and pick the Yugabytedb community dashboard present under the list of Grafana [Dashboards](https://grafana.com/grafana/dashboards/12620).
```

>   - On the Grafana UI, select the `+` icon and choose the import option.

Reviewer: ```suggestion
  - On the Grafana UI, click the `+` button and choose Import.
```

>   ![Grafana import](/images/ce/grafana-add.png)

Reviewer: In the build preview (https://deploy-preview-8924--infallible-bardeen-164bc9.netlify.app/latest/explore/observability/grafana-dashboard/macos/) these aren't indented at the same level as the bulleted text that they belong to. Once you've fixed the indentation of all the text on the page, make sure the images are indented relative to their bullet paragraphs so they align properly.

Author: Hopefully got this one right now.

>   - On the next page, enter the Yugabytedb dashboard ID in the `import via grafana.com` option and click `Load`.

Reviewer: This is a _field_. (Also, capitalization of YugabyteDB.)
```suggestion
  - On the next page, enter the YugabyteDB dashboard ID in the `Import via grafana.com` field and click `Load`.
```

  ![Grafana import](/images/ce/grafana-import.png)

>   - The next screen will require you to provide a name for your dashboard , a unique identifier and the prometheus data source. Make sure to choose the data source you created earlier.

Reviewer: ```suggestion
  - On the next screen, provide a name for your dashboard, a unique identifier, and the Prometheus data source. Be sure to choose the data source you created earlier.
```

  ![Grafana dashboard](/images/ce/graf-dash-details.png)

>   - After clicking import, you can see the new dashboard with various details starting with the status of your Master and Tserver Nodes. The metrics displayed further are categorized by API types (YSQL, YCQL) and by API methods(Insert, Select, Update etc). 

Reviewer: ```suggestion
  - After clicking import, you can see the new dashboard with various details starting with the status of your master and t-server nodes. The metrics displayed are further categorized by API types (YSQL, YCQL) and by API methods(Insert, Select, Update, and so on).
```

  ![Grafana dashboard](/images/ce/graf-server-status.png)

  - Since this example uses the `CassandraKeyValue` workload generator from the [Prometheus Integration](https://docs.yugabyte.com/latest/explore/observability/prometheus-integration/macos/) page, you can see different YCQL related metrics in addition to the Master and Tserver statuses. 

>   Following are some of the YCQL and RocksDB metrics:

Reviewer: ```suggestion

The following are some of the YCQL and RocksDB metrics:
```

>   ![Grafana YCQL](/images/ce/graf-ycql-ops.png)

Reviewer: Since this is serving as a list of examples, you should either make the alt text more descriptive, or work the metric names into the introductory text.

Alternatively, you could keep just the first of the three screenshots, and write something more generic.

Author: I have added more details pertaining to the example and probably made more suitable changes. Please take a took.

  ![Grafana YCQL](/images/ce/graf-ycql-select.png)

  ![Grafana RocksDB](/images/ce/graf-rocksdb.png)

  ## 3. Clean up (optional)

>   Optionally, you can shut down the local cluster created earlier as a prerequisite.

Reviewer: ```suggestion
  Optionally, you can shut down the local cluster you created earlier.
```
